### PR TITLE
Cleanup for Release 2.24.0

### PR DESCRIPTION
**Pull request by jadahiya-MSFT**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

> Summarize the changes, including the goals and reasons for this change. Be sure to call out any technical or behavior changes that reviewers should be aware of.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### Main changes in the PR:

1. Combined changefiles into the CHANGELOG under 2.24.0 release notes
2. Updated integrity hash values for CDN releases

### CODE REVIEW

File: packages/teams-js/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
 
 <!-- Start content -->
 
+## 2.24.0
+
+Thu, 06 Jun 2024 17:06:18 GMT
+
+### Minor changes
+
+- Added a new page property (isBackgroundLoad) for app context. This will be an indicator that the app is being loaded in the background.
```
Comment (AE-MS) [inline packages/teams-js/CHANGELOG.md:13]: >isBackgroundLoad [](http://example.com/codeflow?start=29&length=16)

This should be surrounded with backticks

**Review by AE-MS — CHANGES_REQUESTED** [review]
:clock1:

File: packages/teams-js/CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +24,7 @@ Tue, 07 May 2024 20:54:35 GMT
 - Added `messageChannels.telemetry` and moved existing telemetryPort code into it. Added new `messageChannels.dataLayer` subcapability and added code for interacting with the host data layer.
 - Added `meeting.getMeetingDetailsVerbose` to allow retrieving additional meeting details from supported hosts.
 - Added support for `content` frame context to `liveShare` capability. Live Share SDK works now in Chat/Channel Tab and Collab Stage view contexts with this change.
+  s
```
Comment (AE-MS) [inline packages/teams-js/CHANGELOG.md:27]: ssssssssssssssssssssssssssss

**Review by AE-MS — APPROVED** [review]
:shipit:

**Review by erinha — APPROVED** [review]
:shipit:

**Pull request merged**